rpc/src/rpc.rs

```diff
@@ -3995,6 +4021,30 @@ pub mod rpc_full {
             debug!("get_stake_minimum_delegation rpc request received");
             meta.get_stake_minimum_delegation(config.unwrap_or_default())
         }
+
+        fn get_recent_prioritization_fees(
+            &self,
+            meta: Self::Metadata,
+            pubkey_strs: Vec<String>,
+            config: Option<RpcContextConfig>,
+        ) -> Result<RpcResponse<Vec<u64>>> {
+            debug!(
+                "get_recent_prioritization_fees rpc request received: {:?} pubkeys",
+                pubkey_strs.len()
+            );
+            const MAX_FEE_ACCOUNTS: usize = 42; // TODO: what do we actually want here?
+            if pubkey_strs.len() > MAX_FEE_ACCOUNTS {
+                return Err(Error::invalid_params(format!(
+                    "Too many inputs provided; max {}",
+                    MAX_FEE_ACCOUNTS
+                )));
+            }
+            let pubkeys = pubkey_strs
+                .into_iter()
+                .map(|pubkey_str| verify_pubkey(&pubkey_str))
+                .collect::<Result<Vec<_>>>()?;
```

Comment:
Just making sure I understand this piece of Rust syntax..

The "_" is inferred from the map statement, so it's the type that verify_pubkey returns. The "Vec" says that you should collect these into a vec, and then.. "Result" means to wrap that final vector in a Result? I wasn't aware you could do that, if so!

Reply:
Yes, the type is inferred from the definition of `JsonRpcRequestProcessor::get_recent_prioritization_fees()`
And for the latter: https://doc.rust-lang.org/rust-by-example/error/iter_result.html#fail-the-entire-operation-with-collect, https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect